cmd/oras/root/attach.go

```diff
@@ -44,7 +44,7 @@ type attachOptions struct {
 
 	artifactType string
 	concurrency  int
-	verbose      bool
+	verbose      bool // deprecated
```

Comment:
Should we mark the `verbose` as deprecated based on the golang documentation?

Reply:
Updated.